src/open_samus_returns_rando/files/custom_scenario.lua

```diff
@@ -81,4 +81,12 @@ function Scenario.ShowText()
     Init.tBoxesSeen = textboxes_seen + 1
     GUI.LaunchMessage("#RANDO_STARTING_TEXT_" .. Init.tBoxesSeen, "Scenario.ShowText", "")
   end
+end
+
+function Scenario.LoadNewScenario(target_scenario, target_spawnpoint)
+  bWaitingForScenarioChange = true
+  Game.FadeOut(0.0)
+  Game.FadeOutStream(0.0 * 0.8)
```

Comment:
Since this is explicit, can this be simplified?

Reply:
Yeah, I just replaced the argument with 0.0 and didn't look at it. Changed.